From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: MYIR <support@myirtech.com>
Date: Mon, 7 Sep 2026 16:00:00 +0800
Subject: [PATCH] dts: myir: disable IWDG1 for RT

The MYB watchdog is not serviced after Linux boots, causing OP-TEE to
panic and reset the board after 32 seconds. Disable IWDG1 in the common
MYB device tree so both 1GB and 2GB RT variants remain running.

Signed-off-by: MYIR <support@myirtech.com>
---
 core/arch/arm/dts/myb-stm32mp257x-base.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/myb-stm32mp257x-base.dtsi b/core/arch/arm/dts/myb-stm32mp257x-base.dtsi
index 1111111..2222222 100644
--- a/core/arch/arm/dts/myb-stm32mp257x-base.dtsi
+++ b/core/arch/arm/dts/myb-stm32mp257x-base.dtsi
@@ -354,7 +354,7 @@
 
 &iwdg1 {
 	timeout-sec = <32>;
-	status = "okay";
+	status = "disabled";
 };
 
 &m33_rproc {
-- 
2.34.1

